tests/vmi_configuration_test.go

```diff
@@ -1322,7 +1322,7 @@ var _ = Describe("Configurations", func() {
 				Expect(domXML).To(ContainSubstring("<entry name='serial'>4b2f5496-f3a3-460b-a375-168223f68845</entry>"), "Should have serial-number present")
 			})
 
-			It("should not have serial-number set when not present", func() {
+			It("[test_id:3905]should not have serial-number set when not present", func() {
```

Comment:
This one appears to be test_id:3122

From polarion this test_id appears to be with the name, `should have serial-number set when not present`

Reply:
Fixed